### PR TITLE
Feature test document update

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
## Description

Fixed test_document file, and also deleted some spaces in test_collection. In test_document there are some functions that don't exist at this time like: test_document_replace_match,test_document_update_match,test_document_delete_match,test_document_all,test_document_ids,test_document_keys,test_document_random, so I deleted them. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Reviews

Please identify two developers to review this change

- [x] @ricardo-macrometa
- [x] @dlozina-macrometa
- [x] @koshy-thomas

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added new functional tests that prove my fix is effective or that my feature works
- [x] Existing and new functional tests pass with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: c8/fabric.py [hunk 1]
```diff
@@ -668,10 +668,13 @@ def update_collection_properties(self, collection_name, has_stream=None, wait_fo
 
         :param collection_name: Collection name.
         :type collection_name: str | unicode
+<<<<<<< HEAD
+=======
         :param has_stream: True if creating a live collection stream.
         :type has_stream: bool
         :param wait_for_sync: True if all data must be synced to storage before operation returns.
         :type wait_for_sync: bool
+>>>>>>> upstream/master
```
Comment (koshy-thomas) [inline c8/fabric.py:677]: you will need to remove this probably came when updating branch

File: c8/fabric.py [hunk 2]
```diff
@@ -668,10 +668,13 @@ def update_collection_properties(self, collection_name, has_stream=None, wait_fo
 
         :param collection_name: Collection name.
         :type collection_name: str | unicode
+<<<<<<< HEAD
+=======
```
Comment (koshy-thomas) [inline c8/fabric.py:672]: you will need to remove this probably came when updating branch

File: tests/test_document.py [hunk 3]
```diff
@@ -1,26 +1,21 @@
...
-
 from c8.exceptions import (
-    DocumentCountError,
     DocumentDeleteError,
     DocumentGetError,
-    DocumentInError,
     DocumentInsertError,
     DocumentReplaceError,
     DocumentRevisionError,
     DocumentUpdateError,
```
Comment (koshy-thomas) [inline tests/test_document.py:10]: DocumentUpdateError and DocumentReplaceError might not be required here as we are doing string comparision now for the error. So maybe you can just check if it still needs to be imported from c8.exceptions or not. Also check for other imports which are not required or are not accessed

File: tests/test_document.py [hunk 4]
```diff
@@ -1,26 +1,21 @@
 from __future__ import absolute_import, unicode_literals
-
+import pytest
```
Comment (koshy-thomas) [inline tests/test_document.py:2]: Can you check if we need to import pytest? which statement needs it?

File: c8/fabric.py (hunk 2 above)
Comment (dlozina-macrometa) [reply c8/fabric.py:672]: +1

**Review by dlozina-macrometa — APPROVED** [review]

File: tests/test_document.py (hunk 3 above)
Comment (edgargarciamacrometa) [reply tests/test_document.py:9]: According to Pycharm, all the imports are being used. Regarding the exception type, we have kind of a mix, some functions are making string comparisons, but others are catching the exception type

**Review by koshy-thomas — APPROVED** [review]

**Pull request merged**